R/count-tally.R

```diff
@@ -213,9 +213,9 @@ tally_n <- function(x, wt) {
   }
 
   if (quo_is_null(wt)) {
-    expr(n())
+    new_quosure(expr(n()), asNamespace("dplyr"))
   } else {
-    expr(sum(!!wt, na.rm = TRUE))
+    new_quosure(expr(sum(!!wt, na.rm = TRUE)), quo_get_env(wt))
```

Comment:
This probably should be:

```suggestion
    new_quosure(expr(sum(!!wt, na.rm = TRUE)), baseenv())
```